arches/install/arches-project

```diff
@@ -58,8 +58,24 @@ class ArchesCommand(TemplateCommand):
         elif complete_version[3] in ['alpha','beta','rc']:
             options["arches_version"] = f"dev/{complete_version[0]}.{complete_version[1]}.x"
         options["arches_semantic_version"] = ".".join([str(arches.VERSION[0]), str(arches.VERSION[1]), str(arches.VERSION[2])])
+
         super(ArchesCommand, self).handle('project', project_name, target, **options)
 
+        # need to manually replace instances of {{ project_name }} in some files
+        path_to_project = os.path.join(target, project_name) if target else os.path.join(os.getcwd(), project_name)
+
+        for relative_file_path in ['.yarnrc']:  # relative to app root directory
+            file = open(os.path.join(path_to_project, relative_file_path),'r')
```

Comment:
There's an argument this is a preexisting issue, but if I provide `--destination`, the command fails:

```py
(release) jwalls@Jacobs-MacBook-Pro ~ % mkdir targetDir
(release) jwalls@Jacobs-MacBook-Pro ~ % arches-admin startproject -d targetDir newproject
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/bin/arches-admin", line 186, in <module>
    main()
  File "/Library/Frameworks/Python.framework/Versions/3.12/bin/arches-admin", line 179, in main
    COMMANDS[args.command](args)
  File "/Library/Frameworks/Python.framework/Versions/3.12/bin/arches-admin", line 148, in command_startproject
    cmd.handle(options)
  File "/Library/Frameworks/Python.framework/Versions/3.12/bin/arches-admin", line 133, in handle
    file = open(os.path.join(path_to_project, relative_file_path),'r')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'targetDir/newproject/.yarnrc'
```

Reply:
This is a good catch and a good opportunity to fix the error ++

Reply:
fixed in latest